test/hotspot/jtreg/serviceability/jvmti/vthread/MethodExitTest/libMethodExitTest.cpp

```diff
@@ -500,9 +500,6 @@ VirtualThreadUnmount(jvmtiEnv *jvmti, ...) {
   LOG("\nHit #%d: VirtualThreadUnmount #%d: enabling FramePop for method: %s::%s on virtual thread: %p\n",
```

Comment:
The comment needs to be updated as NotifyFramePop is removed

Reply:
>  think we need a test for the functionality (a new one or update existing)
I've updated this test to request a duplicated `FramePop` event.

> The comment needs to be updated as NotifyFramePop is removed
I've updated this comment accordingly to the change above.